src/vs/workbench/contrib/positronNotebook/browser/useObservedValue.tsx

```diff
@@ -23,12 +22,13 @@ export function useObservedValue<T, M>(observable: ISettableObservable<T>, map?:
 	const [value, setValue] = React.useState(() => typeof map === 'function' ? map(observable.get()) : observable.get());
 
 	React.useEffect(() => {
-		const onObservableChange = Event.fromObservable(observable);
-		const observer = onObservableChange((val) => {
+		const disposable = autorun(reader => {
+			const val = observable.read(reader);
 			setValue(typeof map === 'function' ? map(val) : val);
 		});
-
-		return observer.dispose;
+		return () => {
+			disposable.dispose();
+		}
```

Comment:
Again wondering if we need this change? It is a bit more "obvious" which is good.